packages/ui/src/components/BonnusBanner.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <transition name="fade">
+    <div v-if="player.bonnus" class="info">
```

Comment:
When the player bonus expires, this code makes the banner appear for 1 second saying "X2 Bonus: 1", and then it disappears. I guess it can be fixed like

```suggestion
    <div v-if="player.bonnus >= Date.now()" class="info">
```